### ISSUE TITLE
Duplicate chrom-pos combinations in VCF upload result in cryptic error message

### ISSUE THREAD

**Issue by joerivandervelde** [molgenis/molgenis#4333]
When there are duplicate chrom-pos combinations in the VCF, you get this cryptic error:

```
org.molgenis.data.validation.MolgenisValidationException: Duplicate value 'DJ-I-ir1JivoAZWFvFbsZQ' for unique attribute 'INTERNAL_ID' from entity 'DEMO_GoNL_Cardio' (entity 16839).Duplicate value 'k7OIPu7EgTDhIIt3WHDRwg' for unique attribute 'INTERNAL_ID' from entity 'DEMO_GoNL_Cardio' (entity 16843).Duplicate value 'DZGXZnfImYEQ5PKExxISgQ' for unique attribute 'INTERNAL_ID' from entity 'DEMO_GoNL_Cardio' (entity 16840).Duplicate value 'wc5K1cg69qTK0jMt-LajJA' for unique attribute 'INTERNAL_ID' from entity 'DEMO_GoNL_Cardio' (entity 16841).Duplicate value 'BUO_61dWINJUyrquOvR6gA' for unique attribute 'INTERNAL_ID' from entity 'DEMO_GoNL_Cardio' (entity 16842)
```

This may result from "bad" VCF data, for example:

```
1   201343365   rs59982007  C   T   .   PASS    AC=13;AN=996;DB;GTC=486,11,1;
1   201343398   rs60951162  T   TA  .   PASS    AC=996;AN=996;DB;GTC=0,0,498;
1   201343398   .   A   T   .   PASS    AC=1;AN=996;GTC=497,1,0;
```

It would be nice to report what is really going on to the user so the problem can be fixed.

**Comment by mswertz** [comment]
This seems like a more general issue when autoids are used. We ideally should show the contents of the line/entity instance that produced the fail

**Comment by dennishendriksen** [comment]
This would be solved once we can define a unique constraint on (chrom, pos, ref, alt), see [story in the backlog](http://www.molgenis.org/ticket/4600).

**Comment by dennishendriksen** [comment]
Solved in MOLGENIS 2.0 since we now generate a variant ID based on more columns then chrom/pos.
